beginner_source/blitz/data_parallel_tutorial.py

```diff
@@ -0,0 +1,254 @@
+#         In Model: input size torch.Size([4, 5]) output size torch.Size([4, 2])
+#         In Model: input size torch.Size([2, 5]) output size torch.Size([2, 2])
+#         In Model: input size torch.Size([4, 5]) output size torch.Size([4, 2])
+#         In Model: input size torch.Size([4, 5]) output size torch.Size([4, 2])
+#         In Model: input size torch.Size([4, 5]) output size torch.Size([4, 2])
+#         In Model: input size torch.Size([4, 5]) output size torch.Size([4, 2])
+#         In Model: input size torch.Size([4, 5]) output size torch.Size([4, 2])
+#     Outside: input size torch.Size([30, 5]) output_size torch.Size([30, 2])
+#         In Model: input size torch.Size([4, 5]) output size torch.Size([4, 2])
+#         In Model: input size torch.Size([4, 5]) output size torch.Size([4, 2])
+#         In Model: input size torch.Size([4, 5]) output size torch.Size([4, 2])
+#         In Model: input size torch.Size([4, 5]) output size torch.Size([4, 2])
+#         In Model: input size torch.Size([4, 5]) output size torch.Size([4, 2])
+#         In Model: input size torch.Size([4, 5]) output size torch.Size([4, 2])
+#         In Model: input size torch.Size([2, 5]) output size torch.Size([2, 2])
+#         In Model: input size torch.Size([4, 5]) output size torch.Size([4, 2])
+#     Outside: input size torch.Size([30, 5]) output_size torch.Size([30, 2])
+#         In Model: input size torch.Size([4, 5]) output size torch.Size([4, 2])
+#         In Model: input size torch.Size([4, 5]) output size torch.Size([4, 2])
+#         In Model: input size torch.Size([4, 5]) output size torch.Size([4, 2])
+#         In Model: input size torch.Size([4, 5]) output size torch.Size([4, 2])
+#         In Model: input size torch.Size([4, 5]) output size torch.Size([4, 2])
+#         In Model: input size torch.Size([4, 5]) output size torch.Size([4, 2])
+#         In Model: input size torch.Size([4, 5]) output size torch.Size([4, 2])
+#         In Model: input size torch.Size([2, 5]) output size torch.Size([2, 2])
+#     Outside: input size torch.Size([30, 5]) output_size torch.Size([30, 2])
+#         In Model: input size torch.Size([2, 5]) output size torch.Size([2, 2])
+#         In Model: input size torch.Size([2, 5]) output size torch.Size([2, 2])
+#         In Model: input size torch.Size([2, 5]) output size torch.Size([2, 2])
+#         In Model: input size torch.Size([2, 5]) output size torch.Size([2, 2])
+#         In Model: input size torch.Size([2, 5]) output size torch.Size([2, 2])
+#     Outside: input size torch.Size([10, 5]) output_size torch.Size([10, 2])
+# 
+
+
+######################################################################
+# Summary
+# -------
+# 
+# DataParallel splits your data automatically and sends job orders to multiple
+# models on several GPUs. After each model finishes their job, DataParallel
+# collects and merges the results before returning it to you. It's really awesome!
```

Comment:
Can you remove it's awesome line? You know, it doesn't fit to the tone, I think.